File: CSharp/Algorithm/Palindrome.cs
using static System.Console;
					
public class Program {
	public static void Main() {
		WriteLine(checkPalindrome("ana"));
		WriteLine(checkPalindrome("abba"));
		WriteLine(checkPalindrome("oki"));
	}
	static bool checkPalindrome(string inputString) {
        if(inputString.Length >= 1 && inputString.Length <= 100000) {
            for (var i = 0; i < inputString.Length / 2; i++) if (char.ToLower(inputString[i]) != char.ToLower(inputString[inputString.Length - i - 1])) return false;
			return true;
		}
		return false;
	}
}

//https://pt.stackoverflow.com/q/359775/101

File: JavaScript/Algorithm/Palindrome.js
function checkPalindrome(str) {     
    for(var i = 0; i < str.length / 2; i++) if (str[i] != str[str.length - i - 1]) return false;
    return true;
}
console.log(checkPalindrome("radar"));
console.log(checkPalindrome("reviver"));
console.log(checkPalindrome("palindromo"));

//https://pt.stackoverflow.com/q/315456/101
